src/actions/index.js

```diff
@@ -0,0 +1,20 @@
+import { v4 as generateId } from 'uuid';
+
+export const createBook = ({ category, title }) => ({
+  type: 'CREATE_BOOK',
```

Comment:
**OPTIONAL**: You could define your actions types as constants and use them here and export them to be used on your reducers to minimize errors and make it easier for the next one picking you code so they can easily pick wich are the action types.